charts/kargo/templates/api/configmap.yaml

```diff
@@ -18,16 +18,16 @@ data:
   TLS_CERT_PATH: /etc/kargo/tls.crt
   TLS_KEY_PATH: /etc/kargo/tls.key
   {{- end }}
-  PERMISSIVE_CORS_POLICY_ENABLED: {{ quote .Values.api.enablePermissiveCORSPolicy }}
+  PERMISSIVE_CORS_POLICY_ENABLED: {{ default "false" (quote .Values.api.enablePermissiveCORSPolicy) }}
```

Comment:
This is defaulted to `false` in `values.yaml`. If a user should change that to `""`, this default in the template prevents a problem.

Reply:
Change removed... Go code provides its own default and will also error on values that don't parse as a boolean.